stubs/networkx/networkx/convert.pyi

```diff
@@ -0,0 +1,18 @@
+from _typeshed import Incomplete
+from collections.abc import Iterable
+from typing_extensions import TypeAlias
+
+import numpy
+
+# import scipy
+from networkx.classes.graph import Graph, _EdgePlus, _Node
+
+# this is imported from other stub files
+_Data: TypeAlias = (  # noqa: Y047
```

Comment:
This alias can probably be moved to `stubs/networkx/networkx/classes/graph.pyi`. I don't see it used anywhere else at a glance. Nor a reason to define it in `stubs/networkx/networkx/convert.pyi`